src/ft60x.rs

```diff
@@ -7,151 +7,118 @@ use crate::ft60x_config::FT60xConfig;
 use crate::ringbuf::{RingBuf, RingBufConsumer};
 use crate::Result;
 use failure::format_err;
+use lazy_static::lazy_static;
 
 pub const DEFAULT_PID: u16 = 0x601f;
 pub const DEFAULT_VID: u16 = 0x0403;
 
-pub struct FT60x {
-    vid: u16,
-    pid: u16,
-    context: Context,
+lazy_static! {
+    static ref CONTEXT: Context = Context::new().unwrap();
```

Comment:
did you try what happens if you use this context multiple times?

Reply:
no, but it was used multiple times before as well